portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -125,14 +115,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
    */
     private String getParameter(HttpServletRequest request, String name, String defaultValue) {
         String value = request.getParameter(name);
-        if (value == null) {
+        if (value.equals(null)) {
             return defaultValue;
         }
         return value;
     }
 
     /* Returns number of comments to display */
-    private int getNumberOfComments(HttpServletRequest request) {
+    private int getNumberOfCommentsToDisplay(HttpServletRequest request) {
     // Get the input from the form.
         String numberOfCommentsString = request.getParameter("comments-choice");
```

Comment:
In Java, Javascript, and HTML, I think we all use 2-space indentation unless they break a complete statement.